packages/eslint-plugin/tests/util/getWrappingFixer.test.ts

```diff
@@ -36,7 +36,7 @@ const voidEverythingRule = createRule({
         fix: getWrappingFixer({
           sourceCode: context.sourceCode,
           node,
-          wrap: code => `void ${code}`,
+          wrap: code => `void ${code.replaceAll('wrap', 'wrapped')}`,
```

Comment:
not necessarily a problem, but how is this change related? Seems to just add noise to this PR?

Reply:
Without this replacement, this rule produces infinite fixers :smile:, because the rule listeners, such as `Identifier[name="wrapMe"]`, still match the `wrap*` node.